app/models/job_proxy_dispatcher.rb

```diff
@@ -90,6 +93,53 @@ def dispatch
     _log.info "Complete - Timings: #{t.inspect}"
   end
 
+  def dispatch_container_scan_jobs
+    ret, = Benchmark.realtime_block(:pending_container_jobs) { pending_container_jobs }
+    jobs_by_ems, images_to_ems = ret
+    Benchmark.current_realtime[:container_jobs_to_dispatch_count] = jobs_by_ems.values.reduce(0) { |m, o| m + o.length }
+    concurrent_ems_limit = Settings[:container_scanning][:concurrent_per_ems].to_i
```

Comment:
Note, you can use `Settings.container_scanning.concurrent_per_ems`, I think, so the underlying hash like structure isn't so obvious to clients.